src/v2/schema/message.js

```diff
@@ -0,0 +1,30 @@
+import { gql } from 'apollo-server-express';
+import logger from '../lib/logger';
+export const typeDef = gql`
+type Message {
+  id: String
+  kind: String
+  description: String
+}
+`;
+
+
+export const resolver = {
+  Query: {
+    messages: async (parent, args, { searchModel, req }) => {
+        const messages = [];
+        const disabledClusters = await searchModel.resolveSearchCount({filters:[{ property: 'kind', values: ['cluster']},{ property: 'addon', values: ['search-collector=false']}, {property: 'name', values: ['!local-cluster']}]})
+        if (disabledClusters > 0 ){
+          const id = 'S01'
+          const kind = "information" 
+          const description = 'Currently, search is disabled on some of your managed clusters. Some data might be missing from the console view. See _____ to enable search.'
+          messages.push({
+            id: id,
+            kind: kind,
+            description: description
+          })
```

Comment:
1. We don't need to assign the string to a const. Declaring it inline makes the code easier to read.
2. Let's keep the message text short and concise on the API. Then the UI can add more details. The removed sentences are specific to our web ui client and not relevant to other future clients, like a CLI.

```suggestion
          messages.push({
            id: 'S01',
            kind: 'information',
            description: 'Search is disabled on some of your managed clusters.',
          })
```